### PR TITLE
Lesson 01 python basic

### PR DESCRIPTION
**Pull request by GrzegorzBajorski**
All practice added

Signed-off-by: Grzegorz Bajorski <Grzegorz.Bajorski@tieto.com>

### CODE REVIEW

**Comment by GrzegorzBajorski** [comment]
@gradopaw lesson 1 done - please review

File: students/grzegorz_bajorski/lesson_01_python_basic/car_route.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import math
+
+print('Enter distance which car can cover per day')
+distance = int(input())
+
+print('Enter route lenght')
+lenght = int(input())
```
Comment (gradopaw) [inline students/grzegorz_bajorski/lesson_01_python_basic/car_route.py]: typo lenght -> length

File: students/grzegorz_bajorski/lesson_01_python_basic/car_route.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+days = math.ceil (lenght / distance )
```
Comment (gradopaw) [inline students/grzegorz_bajorski/lesson_01_python_basic/car_route.py]: Not needed whitespace

File: students/grzegorz_bajorski/lesson_01_python_basic/car_route.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import math
```
Comment (gradopaw) [inline students/grzegorz_bajorski/lesson_01_python_basic/car_route.py]: There is no need to import the whole module. If you want to use just one particular function from the module:
from math import ceil

File: students/grzegorz_bajorski/lesson_01_python_basic/total_cost.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+print('Enter cents cost')
+cents = int(input())
+
+print('Enter number of cupcakes')
+cupcakes = int(input())
+
+total =int((dollars * 100 + cents) * cupcakes)
+
+print('total cost in dollars and cents is ' + str(total// 100) + ' ' + str(total % 100))
+
```
Comment (gradopaw) [inline students/grzegorz_bajorski/lesson_01_python_basic/total_cost.py]: Too many empty lines at the end. Should be only one.

File: students/grzegorz_bajorski/lesson_01_python_basic/total_cost.py [hunk 5]
```diff
@@ -0,0 +1,13 @@
+print('Enter dollars cost')
+dollars = int(input())
+
+print('Enter cents cost')
+cents = int(input())
+
+print('Enter number of cupcakes')
+cupcakes = int(input())
+
+total =int((dollars * 100 + cents) * cupcakes)
```
Comment (gradopaw) [inline students/grzegorz_bajorski/lesson_01_python_basic/total_cost.py]: Missing whitespace after '='

**Pull request merged**